lib/node_modules/@stdlib/math/base/special/erfcx/benchmark/benchmark.native.js

```diff
@@ -0,0 +1,57 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var resolve = require( 'path' ).resolve;
+var bench = require( '@stdlib/bench' );
+var randu = require( '@stdlib/random/base/randu' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var pkg = require( '@stdlib/math/base/special/erfcx/package.json' ).name;
+var tryRequire = require( '@stdlib/utils/try-require' );
+
+var erfcx = tryRequire( resolve( __dirname, './../lib/native.js' ) );
+var opts = {
+	'skip': ( erfcx instanceof Error )
+};
```

Comment:
```suggestion
var tryRequire = require( '@stdlib/utils/try-require' );


// VARIABLES //

var erfcx = tryRequire( resolve( __dirname, './../lib/native.js' ) );
var opts = {
	'skip': ( erfcx instanceof Error )
};


```
Please refer other packages to see the conventions, also setup your dev environment by executing `make init`